### DISCUSSION TITLE
cant make a transaction, it always says need an addres

### DISCUSSION THREAD

**Discussion by niagaku909** [woocommerce/woocommerce#52249] [Checkout Flow]
![Gambar WhatsApp 2024-10-23 pukul 09 31 36_9079f037](https://github.com/user-attachments/assets/08aebcb1-dd20-4721-9206-c86d9307eaaf)
![Gambar WhatsApp 2024-10-23 pukul 09 31 36_9079f037](https://github.com/user-attachments/assets/8790e20e-66f7-481c-939f-c2c3410f3805)

why always like that,,,i have try to much for this,,because of this i cant make my store,,my store is not complete
and than one again
![Gambar WhatsApp 2024-10-23 pukul 09 59 13_a7661820](https://github.com/user-attachments/assets/6ca91825-84d5-4ed6-aafa-76f4d3427316)

why you lock all of this,,,,

how i can make my store,how my costumer can checkout,how i can open the lock????

**Comment by juliaamosova** [comment]
Hi @niagaku909! Sorry to hear you are having trouble setting up the checkout. It looks like you created 3 discussions about the same issue. I will keep this one open as it has the most information and close the other two as duplicates:

- https://github.com/woocommerce/woocommerce/discussions/52250
- https://github.com/woocommerce/woocommerce/discussions/52251

Regarding your issue - it looks like you are using a shipping method called "Ninja Express". It is hard to tell what is going on without knowing your payment gateway and shipping setup. Do you use any shipping extensions? What payment gateway do you use? Please provide us with more details about the setup of your store so that we can see how to help further here. Thanks!

**Comment by nielslange** [comment]
Hello @niagaku909 👋

It seems you're facing the same issue as reported in https://wordpress.org/support/topic/company-name-is-required-is-displayed-when-company-is-optional/. I shared a fix for this issue in https://wordpress.org/support/topic/company-name-is-required-is-displayed-when-company-is-optional/#post-18116756. Please let me know if that solved the problem for you.
